test/distributed/org/apache/cassandra/distributed/test/AbstractNodetoolStatusTest.java

```diff
@@ -108,11 +108,11 @@ public void testCommands()
         // test illegal sort by or order
         NodeToolResult invalidSortBy = NODE_1.nodetoolResult("status", "--sort", "not_an_option");
         invalidSortBy.asserts().failure();
-        Assert.assertTrue(invalidSortBy.getStderr().contains("Illegal value for -s/--sort used: 'not_an_option'. Supported values are state, ip, host, load, owns, id, token, rack."));
+        Assert.assertTrue(invalidSortBy.getStdout().contains("Invalid value for option '--sort': expected one of [state, ip, host, load, owns, id, token, rack] (case-sensitive) but was 'not_an_option'"));
```

Comment:
have we changed output from stderr to stdout intentionally?

Reply:
Yes, this behavior has changed for a few commands that use enums or lists of allowed values to validate input arguments. 

The airline prints this error to stderr, while picocli prints it according to the configured error handler. However, most commands print all other errors errors to stdout and with this change now they are aligned.